test/cli-tests/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "cli-tests"
+version = "0.0.0"
+authors = ["Stellar Development Foundation <info@stellar.org>"]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+
+
+[lib]
+crate-type = ["cdylib"]
+doctest = false
+
+[dev-dependencies]
+predicates = "2.1.5"
+soroban-test = {  git = "https://github.com/ahalabs/soroban-tools", rev = "bc1d008" }
```

Comment:
This should probably be the stellar repo, right?

Reply:
Yes, relevant PR is here:

- https://github.com/stellar/soroban-tools/pull/437

We can wait to merge these examples tests until the soroban-tools PR is merged.